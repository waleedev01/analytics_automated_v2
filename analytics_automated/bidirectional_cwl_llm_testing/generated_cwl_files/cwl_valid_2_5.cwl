cwlVersion: v1.2
class: Workflow

# Define the first CommandLineTool step
steps:
  step1:
    run: tool1.cwl  # Reference the CommandLineTool file for step1
    in:
      input1: step1_input1  # Define input for step1
    out:
      output1: step1_output1  # Define output for step1

  # Define the second CommandLineTool step
  step2:
    run: tool2.cwl  # Reference the CommandLineTool file for step2
    in:
      input2: step2_input2  # Define input for step2
    out:
      output2: step2_output2  # Define output for step2

# Define the inputs for the workflow
inputs:
  workflow_input1: string  # Define input for the workflow

# Define the outputs for the workflow
outputs:
  workflow_output1: string  # Define output for the workflow