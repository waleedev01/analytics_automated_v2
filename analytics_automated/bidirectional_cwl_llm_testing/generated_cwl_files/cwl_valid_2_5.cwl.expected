{"is_valid": true, "error": "CWL file is valid.", "file_type": "valid", "file_name": "cwl_valid_2_5.cwl"}